.ci/static-checks.sh

```diff
@@ -309,9 +309,8 @@ static_check_rust_arch_specific()
 		return
 	fi
 
-	find . -type f -name "*.rs"  | egrep -v "target/|grpc-rs/|protocols/" | xargs rustfmt --check > /dev/null 2>&1
+	find . -type f -name "*.rs"  | egrep -v "target/|grpc-rs/|protocols/" | xargs rustfmt --check > /dev/null 2>&1 || die "crate not formatted by rustfmt."
```

Comment:
I guess that works. I think the more common idiom through the file already though is to use a subshell to capture the result, such as (/me grabs random example from earlier in the file):

```sh
    { echo "$name" | grep -q "${check_func_regex}"; ret=$?; }

    [ "$ret" = 0 ] || die "function '$name' is not a check function"
```

/cc @jodh-intel @chavafg

Reply:
I agree it would be good to have a common idiom in the file.

Reply:
@grahamwhaley @chavafg  Updated!